help/C/Help_ch_Transactions.xml

```diff
@@ -1067,28 +1067,112 @@ Author:
+                <row><entry>Flagged for input-transfer account matched.</entry></row>
+                </tbody>
+            </entrytbl>
+          </row>
+          <row>
+           <entrytbl  cols='2' align='left' colsep='1' rowsep='1'>
+                <tbody>
+                  <row><entry><emphasis role="strong">A    </emphasis></entry><entry>-</entry></row>
+                  <row><entry><emphasis role="strong">U+R</emphasis></entry><entry><emphasis>X</emphasis></entry></row>
+                  <row><entry><emphasis role="strong">R    </emphasis></entry><entry>-</entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry><?dbhtml bgcolor="#CCFFCC"?><?dbfo bgcolor="##9FFCC"?>Light Green</entry></row>
+                <row><entry><?dbhtml bgcolor="#FF3333"?><?dbfo bgcolor="#FF3333"?>Bright Red</entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry>Matches an existing transaction - import and update amount.</entry></row>
+                <row><entry>Transaction not matched - not to be imported.</entry></row> <!-- FIXME might not have this quite correct -->
+                </tbody>
+            </entrytbl>
+          </row>
+          <row>
+           <entrytbl  cols='2' align='left' colsep='1' rowsep='1'>
+                <tbody>
+                  <row><entry><emphasis role="strong">A    </emphasis></entry><entry>-</entry></row>
+                  <row><entry><emphasis role="strong">U+R  </emphasis></entry><entry>-</entry></row>
+                  <row><entry><emphasis role="strong">R</emphasis></entry><entry><emphasis>X</emphasis></entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry><?dbhtml bgcolor="#CCFFCC"?><?dbfo bgcolor="#CCFFCC"?>Light Green</entry></row>
+                <row><entry><?dbhtml bgcolor="#FF3333"?><?dbfo bgcolor="#FF3333"?>Bright Red</entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry>Matches an existing transaction - not to be imported.</entry></row>
+                <row><entry>Matches but not reconciled - not to be imported.</entry></row><!--FIXME might not have this quite correct -->
+                </tbody>
+            </entrytbl>
+          </row>
+          <row><entry></entry><entry><?dbhtml bgcolor="#6699CC"?><?dbfo bgcolor="#6699CC"?>Cornflower Blue</entry><entry>Row is selected</entry>
+          </row>
+        </tbody>
+        </tgroup>
+       </table>
+       <para><note>The colors illustrated here are the defaults for GnuCash on a Linux system. If you have altered the CSS
+        or are using another OS the colors may be different.</note>
```

Comment:
I am not  sure, which colors are hardcoded and which are inherited from your desktops default theme. I am now too lazy to look in the source.

Reply:
AFAIK the register colors in GnuCash aren't inherited from the desktop theme as I have changed that without affecting the GnuCash colors at least on Linux Mint and hopefully that should be the case on most Linux distributions.  The default colors are the same on Windows10 ( I ran GnuCash on my wifes Windoze box to check that out). It is more likely to be a problem if someone has customized the GTK CSS to assign different colors. That really can't be controlled, which was why I felt it was worth noting the colors may be different just to alert anyone who does modify the colors.  I selected the colors I used in the table  here by matching the displayed default colors in GnuCash to those from a table of standard internet colors displayed on my monitor in a side by side match. I have tried to track down where the colors are allocated in the code without too much success as yet. They seem to be supplied by gtk_style_context_get() which presumably has defaultsset and also checks any  gtk CSS file. I will keep searching.

Reply:
@DaveC49 Have a look in gnucash-fallback-310.css

Reply:
@Bob-IT Thanks for that- solved the problem.